controllers/storageconsumer/storageconsumer_controller.go

```diff
@@ -361,6 +453,87 @@ func (r *StorageConsumerReconciler) reconcileCephClientRBDNode() error {
 	return nil
 }
 
+func (r *StorageConsumerReconciler) reconcileCephClientCephFSProvisioner() error {
+
+	_, err := ctrl.CreateOrUpdate(r.ctx, r.Client, r.cephClientCephFSProvisioner, func() error {
+		if err := r.own(r.cephClientCephFSProvisioner); err != nil {
+			return err
+		}
+
+		r.cephClientCephFSProvisioner.Spec = rookCephv1.ClientSpec{
+			Caps: map[string]string{
+				"mon": "allow r",
+				"mgr": "allow rw",
+				"osd": fmt.Sprintf("allow rw tag cephfs metadata=* path=%s", r.cephFilesystemSubVolumeGroup.Name),
+			},
+		}
+		return nil
+	})
+
+	if err != nil {
+		r.Log.Error(
+			err,
+			"Failed to update CephClient.",
+			"CephClient",
+			klog.KRef(r.cephClientCephFSProvisioner.Namespace, r.cephClientCephFSProvisioner.Name),
+		)
+		return err
+	}
+
+	cephClientCephFSProvisionerDetails := &ocsv1alpha1.CephResourcesSpec{
+		Kind: r.cephClientCephFSProvisioner.Kind,
+		Name: r.cephClientCephFSProvisioner.Name,
+	}
+	if r.cephClientCephFSProvisioner.Status != nil {
+		cephClientCephFSProvisionerDetails.Phase = string(r.cephClientCephFSProvisioner.Status.Phase)
+	}
+
+	r.updateCephResourceStatus(cephClientCephFSProvisionerDetails)
+
+	return nil
+}
+
+func (r *StorageConsumerReconciler) reconcileCephClientCephFSNode() error {
+
+	_, err := ctrl.CreateOrUpdate(r.ctx, r.Client, r.cephClientCephFSNode, func() error {
+		if err := r.own(r.cephClientCephFSNode); err != nil {
+			return err
+		}
+
+		r.cephClientCephFSNode.Spec = rookCephv1.ClientSpec{
+			Caps: map[string]string{
+				"mon": "allow r",
+				"mgr": "allow rw",
+				"osd": fmt.Sprintf("allow rw tag cephfs *=* path=%s", r.cephFilesystemSubVolumeGroup.Name),
```

Comment:
Providing `path=<>` to the OSD does not make sense - file system "path" is a part of MDS (cap spec) as done below for mds caps.